.github/workflows/release.yaml

```diff
@@ -246,6 +246,21 @@ on:
           branch is considered stale if its version (major, minor, or patch, according to
           `significant-part` in `.ocm/branch-info.yaml`) is older than the current version by at
           least the `supported-versions-count`.
+      github-auth-token-permissions:
+        type: string
+        default: |
+          contents: write
+          pull-requests: read
+          workflows: write
+        description: |
+          Input is passed to `github-auth` action as `permissions` input. The created GitHub access
+          token is used to fetch the release-notes, publish the release commit and delete stale
+          branches (if configured).
+
+          Note: The `workflows: write` permission is required in case there is a concurrent head
+          update of the default branch affecting one of the workflow files. If the release workflow
+          will try to push the changes to a ref, it will be interpreted as a diff towards the default
```

Comment:
for clarification, it should be noted that this will be the case, if the workflow will try to push to a ref !== the default-branch, unless it will rebase prior to doing so, which is not the release-workflows's default behaviour